### PR TITLE
update README for release 0.9.6

### PR DESCRIPTION
**Pull request by nden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Software vs DMS build version map
 
 | jwst tag | DMS build |    Date    |          Notes                           |
 | -------- | --------- | ---------- | -----------------------------------------|
+|  0.9.6   | B7.1.3rc3 | 06/08/2018 | Fourth release candidate for Build 7.1.3 |
```
Comment (hbushouse) [inline README.md]: DMS build should be "rc4"

**Comment by nden** [comment]
Is there a way to cancel the jenkins tests?

**Comment by jhunkeler** [comment]
Yes, but only with an account on the server.

**Comment by nden** [comment]
Tests were terminated, did not fail.

**Pull request merged**